### PR TITLE
[Python 3.6] Solution to challenge 10

### PR DESCRIPTION
**Pull request by Atakyn**

### CODE REVIEW

File: challenge_10/python/Atakyn/c10.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+        if parenthesis < 0 or braces < 0 or brackets < 0:  # someone closed before opening
+            return False
+
+    if parenthesis == 0 and braces == 0 and brackets == 0:  # zero means as many openings as closings
+        return True
+    else:
+        return False
+
+print(valid_closers(input('Input the strings to test: ')))
```
Comment (zmiller91) [inline challenge_10/python/Atakyn/c10.py]: Your solution fails with this test case: `{(})`

Comment (Atakyn) [inline challenge_10/python/Atakyn/c10.py]: > Any characters can appear between any of these

Both open and close, what's wrong with that case?

Comment (moonpatroller) [inline challenge_10/python/Atakyn/c10.py]: @zmiller you guys should fix the readme if you want that case to fail.  It actually reads like {(}) might be ok.

Comment (zmiller91) [inline challenge_10/python/Atakyn/c10.py]: I did not write the problem, nor do I have the ability to change it. The problem is basic, a logical sequence of openers and closers must exist. `{(})` will not compile in any language.

**Pull request closed without merging**